1.Normalize/0.Create-csvs.R

```diff
@@ -0,0 +1,45 @@
+library(tidyverse)
+
+get_db <- function(filepath){
+
+    db <- DBI::dbConnect(RSQLite::SQLite(), filepath)
+    return(db)
+}
+
+## 1. Establish a connection to the database file
+## 2. Load all 4 tables
+## 3. Merge the metadata (Image) table with the other three
+## 4. Print csvs that will be easy to use.
+
+main_function <- function(filepath, platemappath){
```

Comment:
it is great to see this level of modularity! One additional thing you will probably want to update is the hardcoded `Batch2` in lines 31-33.

`filepath` can change, but then the output files are overwritten by having the same filename.

Reply:
also, will `platemappath` eventually be used?